first/frc/index.md

```diff
@@ -6,8 +6,11 @@ title: FRC
 
 Click [here](http://www.firstinspires.org/robotics/frc/what-is-first-robotics-competition) for more.
 
-# __Infinite Recharge 2020__
-This season's game is *FIRST* Infinite Recharge. The teaser and game rules are posted. You can see them [here](https://www.youtube.com/watch?v=1y9PNhVAGmM) and [here](https://www.youtube.com/watch?v=gmiYWTmFRVE). The game manual is currently available for download. The game manual's password is: R!$E2geTHeR#2020.
+## 2020 Challenge: Infinite Recharge!
```

Comment:
Does _FIRST_ call it a challenge or a game? On their website they always refer to it as a game or a season. I would be careful so that you don't confuse people. ("Challenge" might make people think this is the same as the _FIRST_ Tech Challenge.)